.omp/agents/coder.md

```
@@ -0,0 +1,39 @@
+            minLength: 1
+            pattern: '^(?![A-Za-z][A-Za-z0-9+.-]*:)(?![/\\])(?!~(?:[/\\]|$))(?!.*[/\\]{2})(?!.*[/\\]$)(?!\.{1,2}(?:[/\\]|$))(?!.*[/\\]\.{1,2}(?:[/\\]|$))(?!.*(?:%00|\\(?:0|[xX]00|[uU]0000)))(?=\S+$)[^\x00-\x1F\x7F]+$'
+          operation: { type: string, enum: [create, replace, delete] }
+          exact_content: { type: string }
+          patch_intent: { type: string, minLength: 1 }
+    proposed_red_checks: { type: array, items: { type: string, minLength: 1 } }
+    proposed_green_checks: { type: array, items: { type: string, minLength: 1 } }
+    proposed_verification_checks: { type: array, items: { type: string, minLength: 1 } }
+    blockers: { type: array, items: { type: string, minLength: 1 } }
+---
+
+Before acting, read `.Clairvoyance/staff/coder-charter.md`, `.Clairvoyance/library.md`, `AGENTS.md`, and the brief's named spec/plan routes.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)